src/Card/Card.js

```diff
@@ -1,15 +1,42 @@
 import './Card.css';
 import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Card({ name, casting_time, range, duration, desc, handleKnown, handleDelete }) {
+
+  const selectedSpell = {
+    key: name,
+    id: name,
+    name: name,
```

Comment:
glad this worked! i knew we were onto something a few days ago when we were trying this out